gensite.js

```diff
@@ -65,7 +65,7 @@ sh.find(buildPath)
 .filter(function (file) { return file.match(/\.html$/) })
 .forEach(function (f) {
   // replace all RULES.md instances in links with rules.html
-  sh.sed('-i', /"(docs\/|\.\.\/)?RULES(.*?)\.md/g, '"rules$2.html', f)
+  sh.sed('-i', /"(.*?)RULES(.*?)\.md/g, '"rules$2.html', f)
```

Comment:
Is this change backward compatible?

Reply:
Yes, I tested it with the master branch of `standard` and it still generates the links as expected.